[2025-03-27T08:51:42.825+0000] {processor.py:156} INFO - Started process (PID=83964) to work on /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:51:42.825+0000] {processor.py:799} INFO - Processing file /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-27T08:51:42.826+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:51:42.826+0000] {dagbag.py:541} INFO - Filling up the DagBag from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:51:42.829+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:51:42.828+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-27T08:51:42.829+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:51:42.829+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-27T08:51:42.829+0000] {processor.py:811} WARNING - No viable dags retrieved from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:51:42.840+0000] {processor.py:178} INFO - Processing /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-03-27T08:52:32.709+0000] {processor.py:156} INFO - Started process (PID=84517) to work on /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:52:32.710+0000] {processor.py:799} INFO - Processing file /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-27T08:52:32.710+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:52:32.710+0000] {dagbag.py:541} INFO - Filling up the DagBag from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:52:32.712+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:52:32.712+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-27T08:52:32.713+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:52:32.713+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-27T08:52:32.713+0000] {processor.py:811} WARNING - No viable dags retrieved from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:52:32.719+0000] {processor.py:178} INFO - Processing /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2025-03-27T08:53:17.251+0000] {processor.py:156} INFO - Started process (PID=84998) to work on /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:53:17.251+0000] {processor.py:799} INFO - Processing file /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-27T08:53:17.252+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:53:17.252+0000] {dagbag.py:541} INFO - Filling up the DagBag from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:53:17.255+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:53:17.255+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-27T08:53:17.256+0000] {logging_mixin.py:149} INFO - [2025-03-27T08:53:17.255+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-27T08:53:17.256+0000] {processor.py:811} WARNING - No viable dags retrieved from /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-27T08:53:17.266+0000] {processor.py:178} INFO - Processing /home/vscode/miniconda/envs/airflow-env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
